pkg/asset/installconfig/aws/session.go

```diff
@@ -199,9 +199,14 @@ func newAWSResolver(region string, services []typesaws.ServiceEndpoint) *awsReso
 func (ar *awsResolver) EndpointFor(service, region string, optFns ...func(*endpoints.Options)) (endpoints.ResolvedEndpoint, error) {
 	if s, ok := ar.services[resolverKey(service)]; ok {
 		logrus.Debugf("resolved AWS service %s (%s) to %q", service, region, s.URL)
```

Comment:
Shift this down and include the signing region too?

Reply:
the signing region is constant, it is either the region of the cluster or the hard coded default from SDK so i don't think we need that information for debugging.. we can get that information from installer version.